.swiftlint.yml

```diff
@@ -0,0 +1,46 @@
+excluded:
+    - Carthage
+    - Pods/
+    - Tests/
+    - Alicerce.playground/
+reporter: "xcode"
+disabled_rules:
+    - closure_parameter_position
+    - todo
+    - unused_optional_binding
+    - nesting
+    - weak_delegate
+    - large_tuple
+    - type_name
+    - generic_type_name
+    - cyclomatic_complexity
+    - identifier_name
+    - colon
```

Comment:
If I enable this rule we get 70 warnings mainly due to the dictionaries being like `[Key : Value]`
To fix these errors we would need to make them like `[Key: Value]`, since the exception does not work.

I can fix the warnings, but it would imply that we change our style guide.
We can also "humanly" ignore the warnings.

What do ou think?

Reply:
So essentially the rule is still broken 😅 

Fine by me, I prefer to keep it disabled and "manually" check on CR's for now.